### ISSUE TITLE
Wrong desktop icon and titlebar font size

### ISSUE THREAD

**Issue by ottozumkeller** [Abdelrhman-AK/WinPaletter#76]
I just installed version 1.0.7.0 and without doing anything my desktop now looks like this. I assume that something went wrong with the default sizes. Also, I can no longer load the Windows default icon settings into WinPaletter. Also, my title bar height has been changed. I know that this is expected behavior, but WinPaletter should not overwrite my default settings (and even wrong). So how can I restore the Windows default settings?

![image](https://user-images.githubusercontent.com/94012287/219457471-85eb02c0-3088-4756-afb6-77e1b128d779.png)
![image](https://user-images.githubusercontent.com/94012287/219457493-e94fccf2-53b5-47d3-9fa1-1ac940dedec7.png)

**Comment by Abdelrhman-AK** [comment]
- Do you use a high DPI in Windows Settings or not?
- If not, press on a button called `Default Windows` in the form of Metrics & Fonts (With the toggle checked to apply effects)

**Comment by ottozumkeller** [comment]
Yes, my screen uses a resolution of 2160x1350 with a DPI-Scaling of 175%

**Comment by Abdelrhman-AK** [comment]
OK. Here you are the solution: 
- Set DPI to 100%
- Open WinPaletter and open Metrics and Fonts
- Press on `Defaults Windows` button above
- Make sure that the toggle is enabled
- Press on Load and the apply the theme
- Return to Windows Metrics and Fonts
- Make the toggle disabed to prevent further actions
- Re-apply (To save the disabled toggle into registry)
- Reset your DPI to your previous number (175%)

This bug is from Windows when you apply metrics with a high DPI, I will try to make the toggle is disabled by default if the DPI is high. Make sure in most times that the metrics' toggle is disabled to prevent this issue

**Comment by ottozumkeller** [comment]
Thanks. Everything is back to normal. In any case, i've noted that you should be careful with Metrics and Fonts.

**Comment by Abdelrhman-AK** [comment]
I'll try to make the toggle disabed by default when DPI is higher than 100%, and figure out what happens to metrics with high DPI.
